packages/http-client-python/emitter/src/emitter.ts

```diff
@@ -297,7 +296,7 @@ function checkForPylintIssues(outputDir: string) {
     let fileContent = "";
     fileContent = fs.readFileSync(filePath, "utf-8");
     const pylintDisables: string[] = [];
-    const lines: string[] = fileContent.split(os.EOL);
+    const lines: string[] = fileContent.split("/\r?\n/");
```

Comment:
I find returned string of `fs.readFilesSync` may also use line break `\n` instead of `\r\n` in Windows OS which is different with python. So the more safe logic is to use regex expression.